Doc: Remove warnings of textlint (#804)
algolia/docsearch: docs/src/what-is-docsearch.md

## Why?

> We have created DocSearch because we used to scratch our own itch. As

Reviewer: ```suggestion
We have created DocSearch because we are scratching our own itch. As
```

We aren't really using DS anymore for many Algolia projects, but it's still for us in the first place no?

Author: Because we are going way beyond. This sentence states the need and global feeling not the solution. I will keep it. Thanks

## Quick description

> We split DocSearch in a crawler and a frontend library. We run the crawler on

Reviewer: ```suggestion
We split DocSearch into a crawler and a frontend library. We run the crawler on
```

our end every 24h to extract content from your website and push it to an Algolia

> index. You'll then have to add the frontend library to your website to redirect

Reviewer: ```suggestion
index. You can then add the frontend library to your website to search in that index.
```

Reviewer: otherwise you're implying they already need a frontend search

> all the search requests to this index.

Reviewer: ```suggestion
```
